### DISCUSSION TITLE
QTime: Alternative design for format24h

### DISCUSSION THREAD

**Discussion by iliubinskii** [quasarframework/quasar#12163] [General - Components / Directives / etc]
If I am not the only one who thinks that design with 2 circles is ugly, I have the following suggestions:

a) Add "main" slot so that user can replace circle design with anything he wants. I hope this is possible even with minimal interest from comunity.

b) Possible alternative designs:
![Screen3](https://user-images.githubusercontent.com/70687432/150744433-cd3c9a8a-d2df-4c63-a1da-cee8ba8e7055.jpg)

The idea of the second (right) drawing is that:
When arrow points to 12, the circle shows 7 - 17 range, but when you drag arrow to e.g. 15, the circle will show 10 - 20 range, and so on.
Also, there can be fixed ranges selectable by icon buttons:
  Night = 0 - 10
  Day = 6 - 16, or 7 - 17, or 8 - 18 (configurable)
  Evening =13 - 23

**Comment by iliubinskii** [comment]
Another option:
![Screen](https://user-images.githubusercontent.com/70687432/150874539-b5150888-549c-40e6-be68-60e7c8002ddc.jpg)

This approach will let select hours and minutes on the same circle.
Also it is much clearer what you are selecting at the moment when you see long and shirt arrow.

**Reply by TobyMosque** [reply]
https://quasarframework.github.io/quasar-ui-qscroller/demo
